src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/FeatureProfilesController.cs

```diff
@@ -109,173 +114,95 @@ public async Task<IActionResult> Create()
                 return Forbid();
             }
 
-            return View(new FeatureProfileViewModel());
+            var shape = await _displayManager.BuildEditorAsync(new FeatureProfile(), _updateModelAccessor.ModelUpdater, true);
```

Comment:
Can't controllers be used as `IUpdateModelAccessor`? I think it just need to have the interface added to it, and it might provide better results (I don't remember the details).

Reply:
Not sure. Injecting `IUpdateModelAccessor` in the controller's constructor seems to be hows its' done everywhere